zebra/zapi_msg.c

```diff
@@ -854,6 +854,7 @@ void zsend_rule_notify_owner(const struct zebra_dplane_ctx *ctx,
 	stream_putl(s, dplane_ctx_rule_get_seq(ctx));
 	stream_putl(s, dplane_ctx_rule_get_priority(ctx));
 	stream_putl(s, dplane_ctx_rule_get_unique(ctx));
+	stream_putl(s, dplane_ctx_rule_get_vrfid(ctx));
```

Comment:
Doesn't this really belong in the zclient_create_header?

Reply:
I know there's a slot there, but I was concerned about whether someone would mis-interpret ... something, so I kept the new value in the rule-specific encoding?

Reply:
I think it should be about consistency.  The vrf_id should be set properly in the header for the vrf we are talking about.  We should be using VRF_DEFAULT there when the zapi command is VRF free.  I have tried to follow this rule when doing the coding

Reply:
ok, makes sense: I'll push an update...

Reply:
I've pushed an update removing the zclient api change.